FHEM/14_SD_WS.pm

```diff
@@ -1471,6 +1473,21 @@ sub SD_WS_Parse {
                                 }
                               },
     },
+    124 => {
+        # WH31 
+        sensortype => 'WH31',
+        model      => 'SD_WS_124',
+        prematch   => sub { ($rawData,undef) = @_; return 1 if ($rawData =~ /^(30|37)/); },
+        id         => sub {my (undef,$bitData) = @_; return (SD_WS_binaryToNumber($bitData,17,19) + 1);},
+		    channel    => sub {my (undef,$bitData) = @_; return (SD_WS_binaryToNumber($bitData,17,19) + 1);},
+        bat        => sub {my (undef,$bitData) = @_; return substr($bitData,20,1) eq '1' ? 'ok' : 'low';}, #TODO: correct???
```

Comment:
Das Batteriebit wird wahrscheinlich verkehrt herum ausgegeben.

```
        bat        => sub {my (undef,$bitData) = @_; return substr($bitData,20,1) eq '0' ? 'ok' : 'low';},
```

Reply:
Möglich, habe jetzt eher das Problem, dass der Sensor auf eine Antwort warten könnte

Reply:
Kann ich mir nicht vorstellen. Ich glaube kaum, das das Protokoll bidirektional ist.

Reply:
Wundert mich auch:

Diese Beschreibung verwundert mich nur. Im Moment sehe uch alle 5 Minuten was was brauchbares in FHEM. Kann auch gut sein, dass es an meiner Verkabelung liegt.

https://github.com/merbanan/rtl_433/blob/master/src/devices/ambientweather_wh31e.c#L59

Reply:
Den Zeitempfang gibt es wohl nur beim Model WH31E 915 MHz für den amerikanischen Markt. WWV ist wohl ein Zeitzeichensender in den USA siehe https://de.wikipedia.org/wiki/WWV_(Zeitzeichensender).